src/fhir.js

```diff
@@ -384,8 +384,9 @@ class Patient extends FHIRObject {
     }
   }
 
-  findRecords(profile) {
-    const classInfo = this._modelInfo.findClass(profile);
+  findRecords(profile, retrievedDetails) {
```

Comment:
This is a nitpick, but can you rename all occurrences of `retrievedDetails` to `retrieveDetails`? The object is details about an ELM Retrieve expression, not just information that we "retrieved" from somewhere, if that makes sense